### ISSUE TITLE
Credential Guard

### ISSUE THREAD

**Issue by 0xJs** [itm4n/PrivescCheck#49]
Hi itm4n,

Just wanted to let you know, there might be something wrong with the credential guard logic or i'm missing something. Invoke-CredentialGuardCheck might require a little update. Get-WindowsVersion also doesn't exist so it still returns "Credential Guard is not supported on this OS"

On my Windows 11 machine Credential Guard was running (by default on Windows 11) but it wasn't configured.

Check configuration returned false:
```
"CredentialGuard" -match ((Get-ComputerInfo).DeviceGuardSecurityServicesConfigured)
```

Check running returned true:
```
"CredentialGuard" -match ((Get-ComputerInfo).DeviceGuardSecurityServicesRunning)
```

Please close it if you think it isn't an issue, just wanted to let you know and help improve your awesome script!

**Comment by itm4n** [comment]
Hello, 

Thanks for reporting this. I'm already aware of this issue, and it's already fixed. I have several pending changes and updates that I need to push to the main branch.

On Windows 11, Credential Guard is enabled by default without any configuration if all the software and hardware requirements are met. This is why you see it as "running" but not as "configured".

**Comment by itm4n** [comment]
Fixed with commit e1fa7f802c7509ef1512493ae98c834147b00216
